### DISCUSSION TITLE
How can I show brand image?

### DISCUSSION THREAD

**Discussion by krutyosila** [lunarphp/lunar#1569] [Q&A]
How can I show brand image in blade?

**Comment by lguichard** [comment]
Brand Model include the HasMedia trait. You can use Media collection as the Product model. cf : https://docs.lunarphp.io/core/reference/images.html#adding-media-to-models

```php

// Return url
$brand->thumbnail->getUrl()

// Return collection
$brand->getMedia('images') 
```
